### DISCUSSION TITLE
New easAlarm sound feature

### DISCUSSION THREAD

**Discussion by solundev** [HbmMods/Hbm-s-Nuclear-Tech-GIT#653] [Show and tell]
hey there, i edited and created into my mods folder a new sound for easAlarm (because i don't wan't still to bring out a new tape with a new alarm which would take me a lot of time, so i replaced an existing one to test) if any of you like the idea to bring it into oficial updates feel free to ^-
[easAlarm.zip](https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/files/8997927/easAlarm.zip)

**Comment by Shibva** [comment]
ya know, if you can remoive the voices and other stuff and just have the alarm sound that might be considered. thought what do I know
